bot/exts/evergreen/avatar_modification/_effects.py

```diff
@@ -96,6 +96,18 @@ def eight_bitify_effect(image: Image.Image) -> Image.Image:
         image = image.resize((1024, 1024), resample=Image.NEAREST)
         return image.quantize()
 
+    @staticmethod
+    def reverse_effect(image: Image.Image) -> Image.Image:
+        """
+        Flips the image horizontally.
+
+        This is done by just using ImageOps.flip().
+        """
+        image = image.resize((1024, 1024))
```

Comment:
This line isn't needed, since you're already getting the avatar in this size

Reply:
alright